frontend/common/src/components/UserProfile/ProfileSections/DiversityEquityInclusionSection.tsx

```diff
@@ -68,35 +90,140 @@ const DiversityEquityInclusionSection: React.FunctionComponent<{
                   "Label preceding what groups the user identifies as part of, followed by a colon",
               })}{" "}
             </p>{" "}
-            <ul
-              data-h2-font-weight="base(700)"
-              data-h2-padding="base(0, 0, 0, x1)"
+            {indigenousCommunities && indigenousCommunities.length > 0 && (
+              <div
+                data-h2-padding="base(x1, 0, 0, 0)"
+                data-h2-display="p-tablet(flex)"
+                data-h2-align-items="base:(center)"
+                data-h2-justify-content="base:(space-between)"
+              >
+                <ul data-h2-padding="base(0, 0, 0, x1)">
+                  <li>
+                    {intl.formatMessage(
+                      getEmploymentEquityStatement("indigenous"),
+                    )}
+                  </li>
+                  <ul data-h2-padding="base(0, 0, 0, x1)">
+                    <li>
+                      {intl.formatMessage({
+                        defaultMessage: "My Community(ies):",
+                        id: "Mw4ShP",
+                        description:
+                          "Label preceding text listing the communities the user is a member of, followed by a colon",
+                      })}{" "}
+                      {nonLegacyIndigenousCommunities.length > 0
+                        ? nonLegacyIndigenousCommunities
+                            .map((community) => {
+                              return intl.formatMessage(
+                                getIndigenousCommunity(community),
+                              );
+                            })
+                            .join(", ")
+                        : intl.formatMessage(
+                            {
+                              defaultMessage:
+                                "You haven't identified your community yet! <pledgeLink>Please enter here.</pledgeLink>",
+                              id: "1GWQal",
+                              description:
+                                "The text displayed if the user has selected as indigenous but not selected a community yet.",
+                            },
+                            { pledgeLink },
+                          )}
+                    </li>
+                  </ul>
+                </ul>
+                <div
+                  data-h2-display="base(flex) p-tablet(block)"
+                  data-h2-justify-content="base:(center) p-tablet(flex-start)"
+                >
+                  {nonLegacyIndigenousCommunities.map((community) => {
+                    switch (community) {
+                      case IndigenousCommunity.StatusFirstNations:
+                      case IndigenousCommunity.NonStatusFirstNations:
+                        return (
+                          <img
+                            data-h2-float="p-tablet(right)"
+                            data-h2-height="base(4em)"
+                            alt=""
```

Comment:
Should have alt text?  It doesn't have a caption though the text version is nearby.
![image](https://user-images.githubusercontent.com/8978655/217001130-c607ab6f-d874-4291-8e31-351db632c96f.png)

Reply:
@esizer You know a lot about accessibility - do you think we should have alt text for the images here?
![image](https://user-images.githubusercontent.com/49166751/217060035-9d092dd4-f482-4cda-8654-16ec06c07e18.png)

Reply:
I'd say empty alt text is appropriate here (`<img src="..." alt="" />`). These represent the the Indigenous communities which are described  in the [real text next to it making the alt text redundant](https://www.w3.org/WAI/tutorials/images/decision-tree/#:~:text=%E2%80%A6%20and%20it%20shows%20content%20that%20is%20redundant%20to%20real%20text%20nearby.).

Reply:
Thank you!